src/openrct2/network/NetworkBase.cpp

```diff
@@ -49,7 +49,7 @@ using namespace OpenRCT2;
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-constexpr uint8_t kNetworkStreamVersion = 0;
+constexpr uint8_t kNetworkStreamVersion = 1;
```

Comment:
https://github.com/OpenRCT2/OpenRCT2/blob/290b39ade2d70835b001621ac616311b5889abb0/src/openrct2/network/NetworkBase.cpp#L52
This is already at 1 in the current develop branch, please rebase first and then raise it again.

Reply:
It will be at 2 within a few minutes, better to wait a bit (an hour is more than sufficient).